src/test/java/junitparams/ObjectStringificationTest.java

```diff
@@ -1,29 +1,29 @@
 package junitparams;
 
-import static junitparams.internal.Utils.*;
+import junitparams.internal.Utils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static junitparams.internal.Utils.stringify;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.fail;
 
-import org.junit.*;
-import org.junit.runner.*;
-
-import junitparams.internal.*;
-
 @RunWith(JUnitParamsRunner.class)
 public class ObjectStringificationTest {
 
+    private static final String EXAMPLE_NAME = "exampleName";
+    private static final String STRIGIFIED_EXAMPLE_NAME = "[0] exampleName";
```

Comment:
Sure, I personally prefer extracting such values to variables but as always it's a matter of taste :)